src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -147,7 +147,7 @@ private CSVRecord getNextRecord() {
             try {
                 return CSVParser.this.nextRecord();
             } catch (final IOException e) {
-                throw new UncheckedIOException(e.getClass().getSimpleName() + " reading next record: " + e.toString(), e);
+                throw new UncheckedIOException("Error in reading next record: " + e.toString(), e);
```

Comment:
"Error" has a specific meaning in Java, and we did _not_ catch a `java.lang.Error` here. I'll adjust.

Reply:
I feel "Exception reading next record" would be more meaningful.

Reply:
Why not just: "Failed to read next record"?